### DISCUSSION TITLE
Mac -> Windows: Touchpad gestures not supported

### DISCUSSION THREAD

**Discussion by creuter27** [rustdesk/rustdesk#3665] [Q&A]
Thank you for the great work.
It works very well but I have 1 issue that is really bothering me:
I am controlling a Windows desktop client from my Macbook Pro.
I can't scroll or zoom in the Windows client using 2-finger gestures on my Macbook-touchpad. As a workaround, I have to drag the scrollbars in Windows to scroll.

Any help would be much appreciated.

**Comment by rustdesk** [comment]
https://github.com/rustdesk/rustdesk/issues/3744
